src/cljc/rems/common/form.cljc

```diff
@@ -19,6 +19,9 @@
 (defn supports-options? [field]
   (contains? #{:option :multiselect} (:field/type field)))
 
+(defn supports-privacy? [field]
+  (not (contains? #{:label :header} (:field/type field))))
```

Comment:
I wonder if it makes sense to include everything here? I can see that it could be a little confusing if a label is shown but the field it is for is hidden, or an entire section and its header.

Reply:
let's leave it like this for now